### ISSUE TITLE
pod install fails on macOS High Sierra

### ISSUE THREAD

**Issue by biomiker** [GEOSwift/GEOSwift#78]
I'm running macOS High Sierra 10.13.3, XCode 9.2 (9C40b), and cocoapods 1.4.0.  Running 'pod install' for 'GEOSwift' yields the following error.  It seems to be purely related to the SVN export.  I did notice that the geos project has transitioned from SVN to GIT (https://trac.osgeo.org/geos/wiki/CodeRepository).   Has nobody else encountered this?  How can we get our wonderful GEOSwift back under High Sierra?  TIA!!!

```
Installing geos (3.5.0)

[!] Error installing geos
[!] /usr/bin/svn export --non-interactive --trust-server-cert --force https://svn.osgeo.org/geos/tags/3.5.0 /var/folders/n2/5vdk92sj48qgnvyw00jx0q480000gn/T/d20180301-15895-1qd8777

A    /var/folders/n2/5vdk92sj48qgnvyw00jx0q480000gn/T/d20180301-15895-1qd8777
...
A    /var/folders/n2/5vdk92sj48qgnvyw00jx0q480000gn/T/d20180301-15895-1qd8777/include/geos/geom/MultiPolygon.h
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/GeometryCollection.h' forbidden
svn: E200042: Additional errors:
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/Envelope.h' forbidden
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/GeometryFactory.h' forbidden
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/GeometryList.h' forbidden
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/util/ShortCircuitedGeometryVisitor.h' forbidden
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/include/geos/geom/util/PointExtracter.h' forbidden


```

**Comment by mojtabacazi** [comment]
Same issue here. I was thinking about creating a new podspec and change the source to the git repo.

**Comment by vfn** [comment]
@biomiker / @mojtabacazi would you guys like to fix this issue and then create a pull request with the working code?

**Comment by mojtabacazi** [comment]
@vfn I don't think it's fixable here, it seems like pod deosn't like SVN repos.

**Comment by vfn** [comment]
@mojtabacazi you can fix the [geos](https://github.com/GEOSwift/geos) pod and make it point to the git repo.

**Comment by biomiker** [comment]
I wouldn't even know where to start.  :(  

For what it's worth, I feel like a workaround should be possible since the pod install works on older versions of macOS.  And I don't even think it fails on every High Sierra install.  But I don't really know where to start with that either.    Definitely in need of help from experienced contributors/developers.   Please?

**Comment by mojtabacazi** [comment]
This is a problem with SVN 1.9. SVN 1.7 works without problem

**Comment by biomiker** [comment]
Thanks @mojtabacazi ... just curious, it looks like the git does have a 3.5.0 tag so why can't that be used?  Any suggestions on how to install GEOSwift under High Sierra?

**Comment by mojtabacazi** [comment]
@biomiker Like I said if I use 3.5.0, cocoapod will complain about duplicate podspec. Not sure how I can tell it to use one vs the other.

**Comment by biomiker** [comment]
Thanks, I should have gotten that from your previous comment.  So possible solutions at this point are:

1.  Figure out how to update the pod master spec repo to replace the existing 3.5.0 spec with yours 
or
2.  Figure out how to specify in the podfile that GEOSwift 3.5.0 should be pulled from your repo and not the master repo.

Is that about right?

Would one possible workaround be to (perhaps continually) update our local clone of the pod master repo?

**Comment by biomiker** [comment]
Okay so modifying the local clone works!   I'm not sure if this is a good idea or not, but at least it allows the pod to install.  The change is actually very simple:

```
-    "svn": "https://svn.osgeo.org/geos",
+    "git": "https://git.osgeo.org/gitea/geos/geos.git",
```

**Comment by ElnazTaqizadeh** [comment]
I did these changes, but keep getting this error: 

`[!] Unable to find a specification for 'geos'.`

**Comment by vfn** [comment]
I have pushed a new version of `geos` pod, and `GEOSwift` develop branch is now using it. Please let me know if that fixes the issue.

**Comment by ElnazTaqizadeh** [comment]
still returns the same error as @biomiker mentioned.

```
[!] Error installing geos
[!] /usr/bin/svn export --non-interactive --trust-server-cert --force https://svn.osgeo.org/geos/tags/3.5.0 /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16

A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/cmake
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/cmake/modules
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/debian
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/src
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/src/triangulate
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/src/triangulate/quadedge
A    /var/folders/0d/x_1yr12n0l753kb9zf42298w0000gn/T/d20180305-2367-ar3d16/src/algorithm
svn: E175013: Access to '/geos/!svn/rvr/4086/tags/3.5.0/src/triangulate/IncrementalDelaunayTriangulator.cpp' forbidden`
```

**Comment by mojtabacazi** [comment]
@Elnaz can you copy your PodFile here?

**Comment by ElnazTaqizadeh** [comment]
```
 use_frameworks!
 pod 'GEOSwift', :git => 'https://github.com/GEOSwift/GEOSwift.git', :branch => 'develop'
```

**Comment by mojtabacazi** [comment]
@ElnazTaqizadeh  Add the following before `GEOSwift`
`pod 'geos', :git => 'https://github.com/GEOSwift/geos'` 

Run `pod update` (or `pod install`) with `--verbose` flag

**Comment by mojtabacazi** [comment]
@ElnazTaqizadeh TL;DR This should work as is  without problem. Of course you need YOUR_TARGET_NAME with actual target name.

```platform :ios, "10.0"

source 'https://github.com/mojtabacazi/Specs.git'

inhibit_all_warnings!
use_frameworks!

target :YOUR_TARGET_NAME do
pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'
end```

@vfn Your podspec for `geos` is trying to build `geos` for all architecture which takes a long time and almost impossible to debug if something goes wrong during build time.  Mine only runs the `./configure` and let Xcode do the the heavy lifting.

**Comment by mojtabacazi** [comment]
@jwilson85 Copy your podfile here pls

**Comment by jwilson85** [comment]
If I put `source 'https://github.com/CocoaPods/Specs.git'` in there Alamofire give me an error

**Comment by jwilson85** [comment]
So I should put both 
`source 'https://github.com/CocoaPods/Specs.git'`
`source 'https://github.com/mojtabacazi/Specs.git'` 
in there?

**Comment by jwilson85** [comment]
I tried to add both to the pod file and ran a pod install. 
I am back to it hanging at `Installing geos (3.5.1)`

**Comment by jwilson85** [comment]
here is my current pod file 
`
# Uncomment this line to define a global platform for your project
platform :ios, '9.0'

source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/mojtabacazi/Specs.git'

# Comment this line if you're not using Swift and don't want to use dynamic frameworks
use_frameworks!
    
def core_pods 
    
    pod 'Alamofire'
    pod 'SwiftyJSON', :git => 'https://github.com/appsailor/SwiftyJSON.git', :branch => 'swift3'
    pod 'SWXMLHash'
    pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'
    pod 'GoogleAnalytics'   
    pod 'GooglePlaces'
    pod 'GooglePlacePicker'
    pod 'GoogleMaps'
    pod 'Google/SignIn'
    pod 'FacebookLogin'
    pod 'SDWebImage/WebP'
    pod 'Firebase/Core'
    pod 'Firebase/Auth'
    pod 'Firebase/Messaging'
    pod 'SwiftWebViewProgress'
    pod 'FBAnnotationClusteringSwift', :git => 'https://github.com/ribl/FBAnnotationClusteringSwift.git'
    
    # Pods for iOS-Core
    
    #  target 'iOS-CoreTests' do
    #    inherit! :search_paths
    #    # Pods for testing
    #  end
    #
    #  target 'iOS-CoreUITests' do
    #    inherit! :search_paths
    #    # Pods for testing
    #  end
end

target 'iOS-Core' do
    core_pods
end

target 'iOS-Core-beta' do
    core_pods
end

post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
            config.build_settings['SWIFT_VERSION'] = '3.0'
        end
    end
end

`

**Comment by mojtabacazi** [comment]
@jwilson85 configuring `geos` takes a long time that's why I said use `pod update --verbose`  so you can see what's happening.

**Comment by vfn** [comment]
@mojtabacazi when you have a working `geos` podspec, create a PR on https://github.com/GEOSwift/geos and I can update the main pod.

I'm not using `GEOSwift/geos` at the time, so I'm not maintaining them. If you are using and have interest on actively maintaining, I can add you as a collaborator.

**Comment by mojtabacazi** [comment]
@vfn Sure I can fix this issue.

**Comment by jwilson85** [comment]
@mojtabacazi  So I did what you suggested. And I dont understand whats happening. The other pods will say they don't exist when I goto build from xCode. Alternatively, if I copy in the GEOSwift folder and run a pod install everything works.

**Comment by jwilson85** [comment]
Found it 
```bash
[!] /bin/bash -c 
set -e
git submodule update --init --recursive
make

Submodule path 'geos': checked out '6cedf82711369c62c78720ac9966d3cd0f058139'
/Applications/Xcode.app/Contents/Developer/usr/bin/make arch ARCH=armv7 IOS_PLATFORM=iPhoneOS HOST=arm-apple-darwin
echo "Building geos for architecture armv7..."
Building geos for architecture armv7...
cd geos && sh autogen.sh && env \
	CXX=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ \
	CC=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang \
	CFLAGS="-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk -I/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk/usr/include -arch armv7 -I/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/build/armv7/include -miphoneos-version-min=7.0 -fembed-bitcode -Wl,-bitcode_bundle" \
	CXXFLAGS="-stdlib=libc++ -std=c++11 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk -I/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk/usr/include -arch armv7 -I/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/build/armv7/include -miphoneos-version-min=7.0 -fembed-bitcode -Wl,-bitcode_bundle" \
	LDFLAGS="-stdlib=libc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk -L/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/build/armv7/lib -L/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.2.sdk/usr/lib -arch armv7 -miphoneos-version-min=7.0" ./configure --host=arm-apple-darwin --prefix=/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/build/armv7 --disable-static && patch libtool < ../patch-libtool.diff && make clean install 		
Missing autoconf!
Submodule 'geos' (https://github.com/OSGeo/geos.git) registered for path 'geos'
Cloning into '/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/geos'...
env: ./configure: No such file or directory
make[1]: *** [/Users/jeremyw/Library/Caches/CocoaPods/Pods/Release/geos/3.5.1-84cb6/build/armv7/lib/libgeos.dylib] Error 127
make: *** [build_arches] Error 2
```

**Comment by mojtabacazi** [comment]
@jwilson85 That is not using my PodSpec for `geos`. Can you copy your podfile here?

**Comment by jwilson85** [comment]
When using the spec I get 
```bash
[!] /bin/bash -c 
set -e
type -P autoconf &>/dev/null || alias autoconf 'xcrun autoconf'
type -P autoheader &>/dev/null || alias autoheader 'xcrun autoheader'
type -P aclocal &>/dev/null || alias aclocal 'xcrun aclocal'
type -P automake &>/dev/null || alias automake 'xcrun automake'
type -P glibtool &>/dev/null || alias glibtool 'xcrun glibtool'
type -P glibtoolize &>/dev/null || alias glibtoolize 'xcrun glibtoolize'

sh autogen.sh
./configure
./tools/repo_revision.sh

sed -i "" "s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/" include/geos/platform.h
sed -i "" "s/#define HAVE_LONG_INT_64 1/\/\* #undef HAVE_LONG_INT_64 \*\//" include/geos/platform.h

cat <<EOT >> include/geos/platform.h
  #undef ISNAN
  #define ISNAN(x) (std::isnan(x))
EOT

patch capi/geos_c.h <<EOF
  149c149
  < #include <geos/export.h>
  ---
  > #define GEOS_DLL
EOF

/bin/bash: line 2: alias: autoconf: not found
/bin/bash: line 2: alias: xcrun autoconf: not found
```

**Comment by mojtabacazi** [comment]
@jwilson85  Following should solve your problem.

```brew install automake autoconf libtool libxml2 pkg-config
brew link libxml2 ```

**Comment by mojtabacazi** [comment]
@vfn Submitted PR to remove `geos` version restriction

**Comment by himi1** [comment]
@mojtabacazi : This is my current pod file:
```
source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/mojtabacazi/Specs.git'

target 'test123' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for test123
  pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'

end
```

And I did 
```
brew install automake autoconf libtool libxml2 pkg-config
```

The pods installed successfully, but I am now getting the following errors in xcode:
- test123/Pods/GEOSwift/GEOSwift/GEOSwift.h:10:9: Module 'geos' not found
- Could not build Objective-C module 'GEOSwift'

**Comment by jwilson85** [comment]
@himi1  Is your target test123

**Comment by himi1** [comment]
Yes. I created a sample app to display .geo.json on a view controller.

Here's the full podfile:
```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/mojtabacazi/Specs.git'

target 'test123' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for test123
  pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'

  target 'test123Tests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'test123UITests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

**Comment by jwilson85** [comment]
@himi1 Did you also run 
`brew link libxml2 `

**Comment by himi1** [comment]
oh. I did not do that.

It's giving me this warning:
```
Warning: libxml2 is keg-only and must be linked with --force
Note that doing so can interfere with building software.

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/libxml2/bin:$PATH"' >> ~/.bash_profile
````

**Comment by jwilson85** [comment]
try running 
`brew install automake autoconf libtool libxml2 pkg-config` again and then 
`brew link libxml2`

**Comment by himi1** [comment]
I ran the following command to link it now:
```
brew link --force libxml2
```

But xcode errors persist.

**Comment by jwilson85** [comment]
Ya im still having other libs give linker errors. I'm no longer getting a pod install error. which was what I thought was giving me the errors on other libs. Xcode just got to them first I thought and since pods failed it didn't install any.

**Comment by himi1** [comment]
@jwilson85 : What are the errors that you are getting?

**Comment by jwilson85** [comment]
``
ld: framework not found GoogleMaps
clang: error: linker command failed with exit code 1 (use -v to see invocation)
``

I tried to add framework search path that didn't work either.

**Comment by himi1** [comment]
Try deleting ~/Library/Developer/Xcode/DerivedData and then delete Podfile.lock file. Finally, run pod install once again.

**Comment by jwilson85** [comment]
Hey! that seems to have worked. I deleted podfile.lock the pods folder and cleared DerivedData. It's building now. TY!
@himi1

**Comment by himi1** [comment]
@jwilson85 : My pleasure.

Can you copy paste your podfile? I am still unable to fix my xcode errors.

**Comment by jwilson85** [comment]
@himi1  sure
```bash
# Uncomment this line to define a global platform for your project
platform :ios, '9.0'

source 'https://github.com/mojtabacazi/Specs.git'
source 'https://github.com/CocoaPods/Specs.git'

# Comment this line if you're not using Swift and don't want to use dynamic frameworks
use_frameworks!
    
def core_pods 
    
    pod 'Alamofire'
    pod 'SwiftyJSON', :git => 'https://github.com/appsailor/SwiftyJSON.git', :branch => 'swift3'
    pod 'SWXMLHash'
    pod 'GoogleAnalytics'   
    pod 'GooglePlaces'
    pod 'GooglePlacePicker'
    pod 'GoogleMaps'
    pod 'Google/SignIn'
    pod 'FacebookLogin'
    pod 'SDWebImage/WebP'
    pod 'Firebase/Core'
    pod 'Firebase/Auth'
    pod 'Firebase/Messaging'
    pod 'SwiftWebViewProgress'
    pod 'FBAnnotationClusteringSwift', :git => 'https://github.com/ribl/FBAnnotationClusteringSwift.git'
    #pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'

end

target 'iOS-Core' do
    core_pods
end

target 'iOS-Core-beta' do
    core_pods
end

post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
            config.build_settings['SWIFT_VERSION'] = '4.0'
        end
    end
end

```

**Comment by dan-burt** [comment]
I'm having issues installing GeoSwift on Sierra 10.12.6, XCode 9.2, Cocoapods 1.4.0.

Podfile:
```
project 'GeoSwift Sample.xcodeproj'

platform :ios, '9.0'

source 'https://github.com/mojtabacazi/Specs.git'
source 'https://github.com/CocoaPods/Specs.git'

target 'GeoSwift Sample' do
  use_frameworks!

  pod 'GEOSwift', :git => 'https://github.com/mojtabacazi/GEOSwift.git', :branch => 'develop'

end
```

Console reports:

> Analyzing dependencies
> Pre-downloading: `GEOSwift` from `https://github.com/mojtabacazi/GEOSwift.git`, branch `develop`
> Downloading dependencies
> Installing GEOSwift (2.1.3)
> Installing geos (3.6.2)
> [!] /bin/bash -c 
> set -e
> type -P autoconf &>/dev/null || alias autoconf 'xcrun autoconf'
> type -P autoheader &>/dev/null || alias autoheader 'xcrun autoheader'
> type -P aclocal &>/dev/null || alias aclocal 'xcrun aclocal'
> type -P automake &>/dev/null || alias automake 'xcrun automake'
> type -P glibtool &>/dev/null || alias glibtool 'xcrun glibtool'
> type -P glibtoolize &>/dev/null || alias glibtoolize 'xcrun glibtoolize'
> 
> sh autogen.sh
> ./configure
> ./tools/repo_revision.sh
> 
> sed -i "" "s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/" include/geos/platform.h
> sed -i "" "s/#define HAVE_LONG_INT_64 1/\/\* #undef HAVE_LONG_INT_64 \*\//" include/geos/platform.h
> 
> cat <<EOT >> include/geos/platform.h
>   #undef ISNAN
>   #define ISNAN(x) (std::isnan(x))
> EOT
> 
> patch capi/geos_c.h <<EOF
>   149c149
>   < #include <geos/export.h>
>   ---
>   > #define GEOS_DLL
> EOF
> 
> /bin/bash: line 2: alias: autoconf: not found
> /bin/bash: line 2: alias: xcrun autoconf: not found
> 
> 

I have brew installed, and run:
brew install automake autoconf libtool libxml2 pkg-config again and then
brew link --force libxml2

how can i fix this? Thought it shouldn't be this hard with cocoapods!

**Comment by jwilson85** [comment]
@dan-burt did you happen to past the last part of the brew install command into your console. Sorry for the dumb question

**Comment by dan-burt** [comment]
@jwilson85 sorry, the terminal session has already been closed

**Comment by jwilson85** [comment]
@dan-burt just checking to make sure you didn’t past the “again and then” into terminal

**Comment by dan-burt** [comment]
@jwilson85 , sorry I had mis-read your question. No I definitely did not include the "again and then" into the command string

**Comment by ElnazTaqizadeh** [comment]
Every time I update pod for other dependencies the problem appears with different error!This is the new swift compile error:

```
 error: module 'geos' not found
 @import geos;
         ^
 <unknown>:0: error: could not build Objective-C module 'GEOSwift'
```
I tried several ways like clean + build, change the scheme to dependency, delete derived data, none of them worked.

**Comment by Obiwan78** [comment]
Hi Everybody, I have some troubles with this package :
Here the result of my terminal window :

18macbookrj:BuBus alban$ brew install automake autoconf libtool libxml2 pkg-config
Warning: automake 1.16.1_1 is already installed and up-to-date
To reinstall 1.16.1_1, run `brew reinstall automake`
Warning: autoconf 2.69 is already installed and up-to-date
To reinstall 2.69, run `brew reinstall autoconf`
Warning: libtool 2.4.6_1 is already installed and up-to-date
To reinstall 2.4.6_1, run `brew reinstall libtool`
Warning: libxml2 2.9.7 is already installed and up-to-date
To reinstall 2.9.7, run `brew reinstall libxml2`
Warning: pkg-config 0.29.2 is already installed and up-to-date
To reinstall 0.29.2, run `brew reinstall pkg-config`
18macbookrj:BuBus alban$ brew link libxml2 
Warning: libxml2 is keg-only and must be linked with --force

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/libxml2/bin:$PATH"' >> ~/.bash_profile
18macbookrj:BuBus alban$ brew link --force libxml2
Linking /usr/local/Cellar/libxml2/2.9.7... 21 symlinks created

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/libxml2/bin:$PATH"' >> ~/.bash_profile
18macbookrj:BuBus alban$ pod install
Analyzing dependencies
Pre-downloading: `GEOSwift` from `https://github.com/GEOSwift/GEOSwift.git`, branch `develop`
**[!] CocoaPods could not find compatible versions for pod "geos":
  In Podfile:
    GEOSwift (from `https://github.com/GEOSwift/GEOSwift.git`, branch `develop`) was resolved to 2.2.0, which depends on
      geos (= 3.5.0)

None of your spec sources contain a spec satisfying the dependency: `geos (= 3.5.0)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.**

Could you please help me ?

**Comment by macdrevx** [comment]
@Obiwan78 did you try `$ pod repo update`?

**Comment by Obiwan78** [comment]
Yes I do.

**Comment by macdrevx** [comment]
@Obiwan78 based on that screenshot, it looks like you were able to install the SDK. Having a bunch of warnings is, unfortunately, typical at the moment. Do you have any build errors?

**Comment by Obiwan78** [comment]
No, no mistake of build constrcution

**Comment by macdrevx** [comment]
If you just want to ignore the warnings, you can update your podfile:

```
pod 'GEOSwift', :inhibit_warnings => true
pod 'geos', :inhibit_warnings => true
```

After you change you're Podfile, you'll just have to run `$ pod install` again for the changes to take effect.

**Comment by Obiwan78** [comment]
Thanks a lot for your answer. It's good now.

**Comment by macdrevx** [comment]
Hi all, many of these issues have been resolved with GEOSwift 3.0.0 / geos 3.7.0. Give them a try and open new tickets if you run into any problems. I'm closing this one since it's gotten a bit long and hard to follow.
